src/browser/HostInstaller.cpp

```diff
@@ -52,6 +54,7 @@ HostInstaller::HostInstaller()
     // clang-format on
     , TARGET_DIR_FIREFOX("HKEY_CURRENT_USER\\Software\\Mozilla\\NativeMessagingHosts\\org.keepassxc.keepassxc_browser")
     , TARGET_DIR_VIVALDI(TARGET_DIR_CHROME)
+    , TARGET_DIR_BRAVE(TARGET_DIR_CHROME)
```

Comment:
I guess you meant using `TARGET_DIR_BRAVE` instead.

Reply:
@varjolintu Hi! Thanks for the review.

I meant `TARGET_DIR_CHROME` here because I assumed, perhaps wrongly, that Brave functioned on Windows similar to Vivaldi given this comment on #2414: https://github.com/keepassxreboot/keepassxc/issues/2414#issuecomment-468762787

Reply:
Oh, then it's correct.

Reply:
OK. Awesome. Thanks!